backend/testing/tester.py

```diff
@@ -291,6 +289,17 @@ def test(self):
 
             Tester.config_request_rpc(connection)
 
+            # call all helper functions
```

Comment:
the following is for covering code or can it be considered as part of https://github.com/FORTH-ICS-INSPIRE/artemis/issues/41?